### ISSUE TITLE
[Web] the slider freezes after going back and forth

### ISSUE THREAD

**Issue by nakshatrasinghh** [Sharcoux/slider#57]
- Using svg in thumbImage

**Comment by Sharcoux** [comment]
Please try to illustrate your issue with [this snack](https://snack.expo.dev/@sharcoux/sharcoux-slider-demo)

**Comment by nakshatrasinghh** [comment]
https://snack.expo.dev/@nakshatrasinghh/slider1

Here is my snack

**Comment by Sharcoux** [comment]
Can you describe what exactly is the problem?

What is the expected behavior, and what is the current behavior?

**Comment by nakshatrasinghh** [comment]
Do you notice how the slider stops after continuously touching it?


https://user-images.githubusercontent.com/53419293/212381612-c4e4f723-292c-4fea-817f-e3ad0dbdc3d9.mov

**Comment by nakshatrasinghh** [comment]
This behaviour is not happening when the image is not being passed

**Comment by Sharcoux** [comment]
When I check your snack, I don't see the issue you describe...

[slider-bug.webm](https://user-images.githubusercontent.com/797545/212382867-49a2b3f7-4d2d-46db-90f8-69723ac1ddb4.webm)

**Comment by nakshatrasinghh** [comment]
Yes, this issue is not happening on expo, but it happens on react native web view

**Comment by nakshatrasinghh** [comment]
Also, if I run it on my device (expo) after scanning code, this issue still happens

**Comment by Sharcoux** [comment]
Snack uses react-native-web, so it's the same stack. My guess is that your code is trying to control the value of the slider while it is changing. So the slider receive both inputs and has to make a choice between the value that changes when the user moves, and the value that you provide as prop.

Anyway, I can't help more as long as you don't reproduce the bug on expo. The rest is probably related to your implementation that I have no access to.

**Comment by Sharcoux** [comment]
> Also, if I run it on my device (expo) after scanning code, this issue still happens

So it's not on the web? On what OS is it then?

**Comment by nakshatrasinghh** [comment]
Understood, thanks for the help my brother, really means a lot. Cheers and happy coding!

**Comment by nakshatrasinghh** [comment]
> > Also, if I run it on my device (expo) after scanning code, this issue still happens
> 
> So it's not on the web? On what OS is it then?

iOS web

**Comment by nakshatrasinghh** [comment]
> Snack uses react-native-web, so it's the same stack. My guess is that your code is trying to control the value of the slider while it is changing. So the slider receive both inputs and has to make a choice between the value that changes when the user moves, and the value that you provide as prop.
> 
> Anyway, I can't help more as long as you don't reproduce the bug on expo. The rest is probably related to your implementation that I have no access to.

heyy brother, thanks for this, just solved the issue, i was asynchronously changing the value of the text and the slider prop value together, now I made a synchronous call and it works absolutely fine
